ftw/slider/browser/slider.pt

```diff
@@ -13,7 +13,8 @@
         <tal:pane tal:define="internal_link string:${portal/absolute_url}${pane/link};
                               link python: internal_link and pane.link or pane.external_url;">
           <a tal:omit-tag="not:link"
-             tal:attributes="href link">
+             tal:attributes="href link"
+             class="slider-external-link">
```

Comment:
The name might be a bit misleading because it can be either an internal link or an external link.